docs/markdown/06-iterator_generator/05-AsyncIterator.md

```diff
@@ -0,0 +1,50 @@
+<!-- .slide:-->
+
+# Itérables Asynchrones
+
+Un itérable asynchrone est un iterable avec un itérateur dont la méthode next retourne une Promesse de {done, value} au lieu d'un objet {done, value}.
```

Comment:
tu veux pas plutôt mettre : 

Un itérable asynchrone est un iterable avec un itérateur dont la méthode next retourne `new Promise(resolve({done, value}))` au lieu de `{done, value}`.

Reply:
j'essaie de formuler ça comme un type (`Promise<{done, value}>`) sans être trop formel, et `new Promise(resolve => resolve({done, value}))` me paraît donner l'impression qu'on implémentera forcément de cette manière un itérateur asynchrone, sachant qu'il y a en réalité plusieurs façons de le faire.